migration/data.rst

```diff
@@ -0,0 +1,73 @@
+.. _data_migration:
+
+Data Migration
+==============
+
+Migrating from one database to another has traditionally been difficult. There are two traditional approaches: using a dump and restore that takes downtime, or updating application logic to write to two databases and then switch to the new one.
+
+Now there's an easier way. PostgreSQL 10 added logical replication to stream data to a new database even while the existing database continues to sustain application load. It's as if the application automatically writes to two databases rather than one, except with perfect transactional logic.
+
+We have streamlined logical replication on :ref:`Citus Cloud <cloud_overview>` with a process called Citus Warp. To do a warp, we connect the coordinator node of a Citus cluster to an existing database through VPC peering or IP white-listing, and begin replication.
+
+Using Citus Warp
+----------------
+
+Here's the high level overview of the process:
+
+1. Duplicate structure of the schema on a destination Citus cluster
+2. Enable logical replication in the source database
+3. Allow a network connection from Citus coordinator node to source
+4. Make sure no changes will happen to the schema of the source (modifying data is fine, but not structure)
+5. Contact us to run the warp
+
+1. Duplicate schema
+~~~~~~~~~~~~~~~~~~~
+
+The first step in migrating data to Citus is making sure that the schemas match exactly, at least for the tables you choose to migrate. One way to do this is by running ``pg_dump --schema-only`` against the source database. Replay the output on the coordinator Citus node. Another way to is to run application migration scripts against the destination database.
+
+All tables that you wish to migrate must have primary keys. The corresponding destination tables must have primary keys as well, the only difference being that those keys are allowed to be composite to contain the distribution column as well, as described in :ref:`mt_schema_migration`.
+
+Also be sure to distribute tables across the cluster prior to starting a warp so that the data doesn't have to fit on the coordinator node alone.
```

Comment:
Might be worth referencing `create_distributed_table` here just so its clear whats meant with "distribute tables"